CMakeLists.txt

```diff
@@ -20,8 +20,6 @@
 
 cmake_minimum_required(VERSION 3.16)
 
-cmake_policy(SET CMP0091 OLD) # not set MSVC default args
```

Comment:
OLD policies are deprecated by definition. 
- the old policy was: set `/MD` in CMAKE_CXX_FLAGS_<CONFIG> where <CONFIG> is DEBUG, RELEASE, etc.. This allows overriding the option by setting CMAKE_CXX_FLAGS_<CONFIG> manually. However, having to set CMAKE_CXX_FLAGS_<CONFIG> in CMakeLists is not desirable, because that breaks the possibility to set it on the command line.
- new policy: CMake will set the `/MP`/`/MT` flags anyway, regardless of the value of CMAKE_CXX_FLAGS_<CONFIG>. Which flag it takes is controlled by [CMAKE_MSVC_RUNTIME_LIBRARY ](https://cmake.org/cmake/help/latest/variable/CMAKE_MSVC_RUNTIME_LIBRARY.html#variable:CMAKE_MSVC_RUNTIME_LIBRARY)

For now I chose not to set this flag so that the default gets used; that is `/MD`. We previously used `/MT`, but I see no reason for it.